### DISCUSSION TITLE
ctrl+ shift + A doesn't work for reformatting selection

### DISCUSSION THREAD

**Discussion by LR1933** [posit-dev/positron#5531] [Q&A] [answered]
I tried to select the code and open the command palette with Ctrl+Shift+P. But I didn't find the format section, only the format document. I tried to install the styler package, but still no format section. Am I doing something wrong?

![image](https://github.com/user-attachments/assets/22d05c1e-407e-4bf4-84a1-9c78d5092274)

This is the version information:

> Positron Version: 2024.12.0 (system setup) build 41
> Code - OSS Version: 1.93.0
> Commit: https://github.com/posit-dev/positron/commit/39536c265d6828a68291984d0d359170316d055e
> Date: 2024-11-18T02:47:00.983Z
> Electron: 30.4.0
> Chromium: 124.0.6367.243
> Node.js: 20.15.1
> V8: 12.4.254.20-electron.0
> OS: Windows_NT x64 10.0.22631

This is my keybindings.json (no setting):

> [
> ]
> 

This is my settings.json:

```
{
"workbench.colorTheme": "Default Positron Dark",
"quarto.visualEditor.spelling": true,
"js/ts.implicitProjectConfig.checkJs": true,
"python.analysis.diagnosticMode": "workspace",
"git.confirmSync": false,
"explorer.confirmDelete": false,
"problems.defaultViewMode": "table",
"problems.showCurrentInStatus": true,
"editor.autoClosingBrackets": "beforeWhitespace",
"editor.autoClosingDelete": "never",
"positron.r.pipe": "%>%",
"editor.minimap.enabled": true,
"editor.minimap.maxColumn": 200,
"application.experimental.positronPlotsInEditorTab": true,
"rstudio.keymap.enable": true
}
```

**Comment by juliasilge** [comment, marked as answer]
Turns out you are not the only one with this problem! In #5534 we also see folks who are not able to use the formatter. Can you take a look at the [questions I ask here](https://github.com/posit-dev/positron/issues/5534#issuecomment-2504822725), and see what you observe?
